cmd/kubelet/app/server.go

```diff
@@ -168,7 +168,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			if err != nil {
 				klog.Fatal(`"help" flag is non-bool, programmer error, please correct`)
 			}
-			if help {
+			if help || cleanFlagSet.NFlag() == 0 {
```

Comment:
The error message suggests `Cannot connect to the Docker daemon at unix:///var/run/docker.sock`, maybe we should look at this one.

```
F0312 10:21:53.879084   27233 server.go:269] failed to run Kubelet: failed to get docker version: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
goroutine 1 [running]:
k8s.io/kubernetes/vendor/k8s.io/klog/v2.stacks(0xc000120001, 0xc00026a0d0, 0xc4, 0xd0)
```

Reply:
/hold

Reply:
this may need new pr ,it not main error for this.

Reply:
> when running `kubelet` it should be the same as `kubelet --help`
ohh, i got your point,  `kubelet` it means running with the default args, should not be like `kubelet --help`